src/CitizenProject/CitizenProjectFactory.php

```diff
@@ -75,10 +78,18 @@ public function createFromCitizenProjectCreationCommand(CitizenProjectCreationCo
             $command->problemDescription,
             $command->proposedSolution,
             $command->requiredMeans,
-            $command->getCommittee(),
+            $command->getCommitteeSupports()->toArray(),
             $command->getAddress() ? $this->addressFactory->createFromNullableAddress($command->getAddress()) : null
         );
 
+        $committeeSupports = $citizenProject->getCommitteeSupports()->toArray();
+        foreach ($committeeSupports as &$object) {
```

Comment:
Ok so the problem is still coming from the CitizenProject. How can it have a polymorphic collection in the first place?